app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1036,11 +1036,26 @@ class PatientRepositoryAndroidTest {
     assertEquals(creationTimeOf2ndBp, recentPatientAfter2ndBp.lastBp!!.createdAt)
   }
 
+  @Test
+  fun verify_deleted_bps_are_not_picked_up() {
```

Comment:
This test doesn't specify that the bps should not be picked up for what?
Can you rename it to  -> `verify_deleted_bps_are_not_included_when_fetching_recent_patients`
Also, do the same for the test `verify_BP_from_all_facilities_are_picked_up(0`